pkg/model/config/config.go

```diff
@@ -125,11 +127,45 @@ func (c Config) HasGroup(group string) bool {
 	return false
 }
 
+// CRDVersionCompatible returns true if crdVersion can be added to the existing set of CRD versions.
+func (c Config) CRDVersionCompatible(crdVersion string) bool {
```

Comment:
```suggestion
func (c Config) IsCRDVersionCompatible(crdVersion string) bool {
```
Could we use is in the bool ones? Shows more intuitive knows that will return a bool